docker/code-check.Dockerfile

```diff
@@ -0,0 +1,7 @@
+# Image for all tasks related to static code analysis in Async-Profiler
+FROM public.ecr.aws/docker/library/amazoncorretto:11-alpine-jdk
+
+ADD https://raw.githubusercontent.com/llvm/llvm-project/refs/tags/llvmorg-20.1.7/clang-tools-extra/clang-tidy/tool/clang-tidy-diff.py /usr/bin/
```

Comment:
That is better. However, tags are still mutable. Specific commit id would be ideal.